test/goldens/build_file_web_application

```diff
@@ -28,3 +31,22 @@ dart_web_application(
         ":silly_monkey",
     ],
 )
+dev_server(
+    name = "main_web_dartium_serve",
+    deps = _PUB_DEPS + [
+        ":silly_monkey",
+    ],
+    data = glob(["web/**"]),
+)
+dart_ddc_bundle(
+    name = "main_web_ddc_bundle",
+    entry_library = "web/main.dart",
```

Comment:
This isn't a part of `:silly_monkey`. I think we need to materialize an additional dart_library which has as its sources `glob(["web/**"])`? Setting up the correct dependencies would also require reading web/main.dart and examining its imports or something along those lines....